### PR TITLE
feat: display one to one conversation user membership

### PR DESCRIPTION
**Pull request by trOnk12**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

- integrate the user type from Kalium

### Attachments (Optional)

image
----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt [hunk 1]
```diff
@@ -163,4 +167,14 @@ class ConversationListViewModel @Inject constructor(
...
         }
     }
+
+    private fun mapUserType(userType: UserType): Membership {
+        return when (userType) {
+            UserType.GUEST -> Membership.Guest
+            UserType.FEDERATED -> Membership.Federated
+            UserType.EXTERNAL -> Membership.External
+            UserType.INTERNAL -> Membership.None
+            else -> {  throw IllegalStateException("Unknown UserType") }
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt:177]: Why this else?

IDE might complain a bit, but the compiler knows that this `when` is exhaustive and there's no need for it.

**Review by vitorhugods — APPROVED** [review]
🚀🚀

Awesome to see this done 😄

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-543/3/) **failed**.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -309,4 +309,7 @@
     <string name="content_description_ping_everyone">Ping everyone</string>
     <string name="label_message_sent_failure">Message could not be sent.</string>
     <string name="label_try_again">Try again</string>
+    <string name="content_description_ping_everyone">Ping everyone</string>
```
Comment (gongracr) [inline app/src/main/res/values/strings.xml:312]: Why is this string duplicated?

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-543/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3218835)(3218835)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3218835)(3218835)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-543/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-543/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3218956)(3218956)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3218956)(3218956)-dev-release.apk)

**Pull request merged**